modules/distr-tracing-config-security-ossm-cli.adoc

```diff
@@ -0,0 +1,93 @@
+////
+This module included in the following assemblies:
+service_mesh/v2x/ossm-reference-jaeger.adoc
+////
+:_content-type: PROCEDURE
+[id="distr-tracing-config-security-ossm-cli_{context}"]
+= Configuring distributed tracing security for service mesh from the command line
+
+You can modify the Jaeger resource to configure {JaegerShortName} security for use with {SMproductShortName} from the command line using the `oc` utility.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.  If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+* The {SMProductName} Operator must be installed.
+* The `ServiceMeshControlPlane` deployed to the cluster.
+* You have access to the OpenShift CLI (oc) that matches your OpenShift Container Platform version.
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
++
+[source,terminal]
+----
+$ oc login https://<HOSTNAME>:6443
+----
++
+. Change to the project where you installed the control plane, for example istio-system.
++
+[source,terminal]
+----
+$ oc project istio-system
+----
++
+. Run the following command to edit the Jaeger custom resource file, where `jaeger.yaml` is the name of your Jaeger custom resource.
++
+[source,terminal]
+----
+$ oc edit -n tracing-system -f jaeger.yaml
+----
++
+. Edit the `Jaeger` custom resource file to add the `htpasswd` configuration as shown in the following example.
+
+* `spec.ingress.openshift.htpasswdFile`
+* `spec.volmes`
+* `spec.volumeMounts`
```

Comment:
See my comments below on this list, and whether callouts might better server customers.